Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -733,6 +730,35 @@ private class PopupBlockingSettings: Setting {
     }
 }
 
+private class SaveLoginSettings: Setting {
```

Comment:
Also, maybe introduce some naming consistency here. Either all <name>Settings, all <name>Setting or all <name>.

I know this is cleaning up other peoples code, but it just niggles, you know?